modules/installation/proc-rhdh-deploy-eks-operator.adoc

```diff
@@ -1,12 +1,20 @@
-[id='proc-rhdh-deploy-eks-using-operator_{context}']
+// Module included in the following assemblies
+// assembly-install-rhdh-eks.adoc
 
-= Deploying {product} on {eks-name} ({eks-short}) using the Operator
+[id='proc-rhdh-deploy-eks-operator_{context}']
+= Installing {product-short} on {eks-short} with the Operator
 
-You can deploy the {product-short} on {eks-short} using the {product} Operator with or without https://olm.operatorframework.io[Operator Lifecycle Manager (OLM) framework]. Following that, you can proceed to install your {product-short} instance in {eks-short}.
+You can install the {product} Operator with or without the Operator Lifecycle Manager (OLM) framework.
+
+.Additonal resources
```

Comment:
I believe we can omit the additional resources tag here. Typically, additional resources are added at the end of a section, but here the Installation on EKS section is just starting.